outputs.tf

```diff
@@ -19,9 +19,14 @@ output "backend_services" {
   value       = google_compute_backend_service.default
 }
 
-output "external_ip" {
```

Comment:
Let's also preserve the `external_ip` output for backwards-compatibility. Just make it return the first address which is available.

Reply:
Agreed - not touching the `external_ip` variable.
However, I'd prefer to output new variable - something like `external_ipv6_address`.
That way, both addresses are available for output.